src/hooks/useRegisterBirthDate.ts

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+
+const useRegisterBirthDate = () => {
+    const [selectedDate, setSelectedDate] = useState<Date>();
+
+    const isValidBirthDate = () => selectedDate ? selectedDate < new Date() : false
+
+    const getFormattedBirthDate = (): string => {
+        if (selectedDate) {
+            const day = selectedDate.getDate()
+            const month = selectedDate.getMonth()
+            const year = selectedDate.getFullYear()
+            const dayPad = day < 10 ? '0' : ''
+            const monthPad = month < 10 ? '0' : ''
```

Comment:
remember the thing about the [magic numbers ](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,see%20List%20of%20file%20signatures)